.circleci/config.yml

```diff
@@ -351,6 +376,7 @@ workflows:
               ignore:
                 - env/dev1
                 - env/dev2
+                - master
```

Comment:
Well spotted.

I see that these 3 jobs are mutually exclusive:
- docker_image_build
- docker_push_dev
- docker_push_tag